src/plugin.validation.dns.godaddy/GodaddyArgumentsProvider.cs

```diff
@@ -0,0 +1,21 @@
+﻿using Fclp;
+using PKISharp.WACS.Configuration;
+
+namespace PKISharp.WACS.Plugins.ValidationPlugins
+{
+    public class GodaddyArgumentsProvider : BaseArgumentsProvider<GodaddyArguments>
+    {
+        public override string Name => "Godaddy";
+
+        public override string Group => "Validation";
+
+        public override string Condition => "--validationmode dns-01 --validation dreamhost";
+
+        public override void Configure(FluentCommandLineParser<GodaddyArguments> parser)
+        {
+            _ = parser.Setup(o => o.ApiKey)
+                .As("apiKey")
```

Comment:
Please use lowercase command line parameters to be consistent with the rest of them.